packages/core/js-client/src/services/_aqua/single-module-srv.ts

```diff
@@ -14,149 +14,21 @@
  * limitations under the License.
  */
 
-/**
- * This compiled aqua file was modified to make it work in monorepo
- */
-import { CallParams, ServiceImpl } from "@fluencelabs/interfaces";
-
 import { registerService } from "../../compilerSupport/registerService.js";
 import { FluencePeer } from "../../jsPeer/FluencePeer.js";
 import { Srv } from "../SingleModuleSrv.js";
 
-// Services
-
-export interface SrvDef {
-  create: (
-    wasm_b64_content: string,
-    callParams: CallParams<"wasm_b64_content">,
-  ) =>
-    | { error: string | null; service_id: string | null; success: boolean }
-    | Promise<{
-        error: string | null;
-        service_id: string | null;
-        success: boolean;
-      }>;
-  list: (callParams: CallParams<null>) => string[] | Promise<string[]>;
-  remove: (
-    service_id: string,
-    callParams: CallParams<"service_id">,
-  ) =>
-    | { error: string | null; success: boolean }
-    | Promise<{ error: string | null; success: boolean }>;
-}
-
 export function registerSrv(
```

Comment:
doc

Reply:
As far as I can see this was ignored as well